### ISSUE TITLE
Interleaving output only

### ISSUE THREAD

**Issue by Poshi** [marcelm/cutadapt#164]
This is just a suggestion. I saw that you added an option for processing interleaved data. The option seems to be tied to both input and output. I think that it would make sense to untie input and output. Many times the input will be from two raw files that will be prepared with cutadapt and feeded with a pipe to an aligner, so you need paired (non-interleaved) input and interleaved output.
And mabye, some times, you need to read from an interleaved input and generate two files, one per end.

**Comment by marcelm** [comment]
Thanks for letting me know. When I implemented this, I was aware of this restriction, but decided to wait until someone actually requested this feature. I’ll leave this issue open, but unless you or someone else has an actual use case for it, I’ll give it lower priority for now.

**Comment by marcelm** [comment]
I’m adding this as a note to myself since I don’t have time to implement this right now, but have thought about the user interface (which is to me always the most difficult part). I think the easiest way is to implement it by looking at the number of file names given, for input and output separately: If it’s two, then don’t interleave. If it’s one, then do.

Both input and output in two files (no interleaving):
```
cutadapt -o out.1.fq -p out.2.fq in.1.fq in.2.fq
```
Both interleaved:
```
cutadapt --interleaved -o out.fq in.fq
```
Input interleaved, output two files:
```
cutadapt --interleaved -o out.1.fq -p out.2.fq in.fq
```
Output interleaved, input two files:
```
cutadapt --interleaved -o out.fq in.1.fq in.2.fq
```

**Comment by marcelm** [comment]
Implemented in cutadapt 1.11, released just now.

**Comment by Poshi** [comment]
Thanks!
